### PR TITLE
Small correction, typo fix in backward_cfi

### PR DESCRIPTION
**Pull request by deepak0414**
adding ssamoswap can cause access violation too.
few typo fixes and cosmetic changes.

Signed-off-by: Deepak Gupta <debug@rivosinc.com>

### CODE REVIEW

File: cfi_backward.adoc [hunk 1]
```diff
@@ -420,13 +420,13 @@ The following faults may occur:
 .. Instructions fetch causes a page fault
 . if the accessed page is not a shadow stack page
 .. `ssamoswap` and `sspush` cause a store/AMO access fault
-.. `sspop` causes a load access fault
+.. `sspop` and `ssamoswap` cause a load access fault
```
Comment (ved-rivos) [inline cfi_backward.adoc:423]: ssamoswap as noted in line 422 causes a store/AMO access fault.

**Review by ved-rivos — CHANGES_REQUESTED** [review]
ssamoswap causes store/amo access fault and not load access fault. That change should be removed. Please provide the update PR.

File: cfi_backward.adoc (hunk 1 above)
Comment (deepak0414) [reply cfi_backward.adoc:423]: Thanks for pointing that out.
Let me fix it.

Comment (deepak0414) [reply cfi_backward.adoc:423]: fixed it. please see new commit on same branch.

**Pull request merged**